src/components/sbb-journey-summary/sbb-journey-summary.scss

```diff
@@ -0,0 +1,60 @@
+.sbb-journey-summary {
+  @include text-s--regular;
+
+  padding: var(--sbb-spacing-responsive-s);
+  display: flex;
+  flex-direction: column;
+}
+
+.sbb-journey-summary__via-block {
+  display: flex;
+
+  .sbb-journey-summary__via-text {
```

Comment:
@k-luise is this nesting nessesary? I just mention it because of ["Use lowest specificity possible"](https://github.com/lyne-design-system/lyne-components/blob/master/CODING_STANDARDS.md#use-lowest-specificity-possible) coding guidelines.